src/Components/Members/MembersCreateEdit.js

```diff
@@ -0,0 +1,316 @@
+              />
+              <h4 className="title">Image</h4>
+              <section className="form">
+                <div
+                  {...getRootProps({ className: 'dropzone' })}
+                  style={{ margin: '10px auto' }}>
+                  <input
+                    {...getInputProps({
+                      id: 'image',
+                      name: 'image',
+                      display: 'block',
+                    })}
+                  />
+                  <p style={{ textAlign: 'center' }}>
+                    Drag and drop some files here, or click to select files
+                  </p>
+                  <aside className="thumbs-container">{thumbs}</aside>
+                </div>
+              </section>
+              <ErrorMessage
+                component={() => (
+                  <Alert severity="warning">{errors.image}</Alert>
+                )}
+                name="image"
+              />
+              <h4 className="title">facebookUrl</h4>
+              <Field
+                fullWidth
+                component={TextField}
+                error={Boolean(touched.facebookUrl && errors.facebookUrl)}
+                id="facebookUrl"
+                label="facebookUrl"
+                name="facebookUrl"
+                placeholder="facebookUrl"
+                sx={{ margin: '20px auto', width: '80%', display: 'flex' }}
+                type="text"
+                value={initialValues.facebookUrl}
+                variant="outlined"
+                onChange={(e, value) => facebookChange(e, value)}
+              />
+              <ErrorMessage
+                component={() => (
+                  <Alert severity="warning">{errors.facebookUrl}</Alert>
+                )}
+                name="facebookUrl"
+              />
+              <h4 className="title">linkedinUrl</h4>
+              <Field
+                fullWidth
+                component={TextField}
+                error={Boolean(touched.linkedinUrl && errors.linkedinUrl)}
+                id="linkedinUrl"
+                label="linkedinUrl"
+                name="linkedinUrl"
+                placeholder="linkedinUrl"
+                sx={{ margin: '20px auto', width: '80%', display: 'flex' }}
+                type="text"
+                value={initialValues.linkedinUrl}
+                variant="outlined"
+                onChange={linkedInChange}
+              />
+              <ErrorMessage
+                component={() => (
+                  <Alert severity="warning">{errors.linkedinUrl}</Alert>
+                )}
+                name="linkedinUrl"
+              />
+              <Button
+                className="submit-btn"
+                type="submit"
+                variant="contained"
+                onClick={() => handleClick()}>
+                Send
+              </Button>
+            </Form>
+          </Card>
+        );
+      }}
+    </Formik>
+  );
+}
+
+export default MembersCreateEdit;
```

Comment:
why do you have a member form if you need to add a news form?

Reply:
me equivoque y estan los dos formularios pusheados en la misma rama, el de news y el de members